test/jmap/identities/get_identities_with_sort_order_test.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:dio/dio.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http_mock_adapter/http_mock_adapter.dart';
+import 'package:jmap_dart_client/http/http_client.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/properties/properties.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/identities/get/get_identity_method.dart';
+import 'package:jmap_dart_client/jmap/identities/get/get_identity_response.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/jmap/jmap_request.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
+
+void main() {
+  group('test to json get identity method', () {
```

Comment:
@sherlockvn please add the test for the case:
- increase `sortOrder` of default Identity 
- create new default Identity 
All in one request